From ced34621c7e79bf119f328740b761f77beaecf10 Mon Sep 17 00:00:00 2001
From: Yumi Yukimura <me.cafebabe@gmail.com>
Date: Tue, 13 Feb 2024 00:50:59 +0800
Subject: [PATCH 138/157] input: touchscreen: Makefile: Move
 goodix_driver_gt9886 before fts_521

* This driver needs to be initialized before fts_521, in order to not to break it

Change-Id: I46d6b1ee1c17aba8965a1d95a845c5c662124806
---
 drivers/input/touchscreen/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/Makefile b/drivers/input/touchscreen/Makefile
index a037dcbd0faa..aeb234d33182 100644
--- a/drivers/input/touchscreen/Makefile
+++ b/drivers/input/touchscreen/Makefile
@@ -120,10 +120,10 @@ obj-$(CONFIG_INPUT_PRESS_NDT) += ndt_button.o
 obj-$(CONFIG_TOUCHSCREEN_FOCALTECH_3658U) += focaltech_3658u/
 obj-$(CONFIG_TOUCHSCREEN_FTS_MI)	+= focaltech_touch_mi/
 obj-$(CONFIG_TOUCHSCREEN_FOCALTECH) += focaltech_spi/
+obj-$(CONFIG_TOUCHSCREEN_GOODIX_GTX8) += goodix_driver_gt9886/
 obj-$(CONFIG_TOUCHSCREEN_ST_FTS_V521) += fts_521/
 obj-$(CONFIG_TOUCHSCREEN_ST_FTS_V521_SPI) += fts_spi/
 obj-$(CONFIG_TOUCHSCREEN_GOODIX_BRL) += goodix_9916/
-obj-$(CONFIG_TOUCHSCREEN_GOODIX_GTX8) += goodix_driver_gt9886/
 obj-$(CONFIG_TOUCHSCREEN_HIMAX_CHIPSET) += hxchipset/
 obj-$(CONFIG_TOUCHSCREEN_NT36523_HOSTDL_SPI) += nt36523/
 obj-$(CONFIG_TOUCHSCREEN_NT36xxx_HOSTDL_SPI) += nt36672c/
-- 
2.48.1

